server_development/topics/providers.adoc

```diff
@@ -391,6 +438,8 @@ Once you have a JAR file with a descriptor and the scripts you want to deploy, y
 
 ==== Using {project_name} Administration Console to upload scripts
 
+NOTE: Ability to upload scripts through the admin console is deprecated and will be removed in a future version of {project_name}
```

Comment:
+ NOTE Ability to ...

Reply:
"NOTE:" is a asciidoc syntax to render a note block, so should be NOTE:, not NOTE